doc/source/releases/16.07.rst

```diff
@@ -89,6 +51,28 @@ Highlighted Enhancements
 * Add `EDAM <http://edamontology.org/page>`__ operations to tools and expose in API
   (thanks to `@mvdbeek <https://github.com/mvdbeek>`__, `@nsoranzo <https://github.com/nsoranzo>`__ and `@odoppelt <https://github.com/odoppelt>`__.)
   `Pull Request 2379`_
+* Add support to PDBQT molecule datatype
+  (thanks to `@leobiscassi <https://github.com/leobiscassi>`__.)
+  `Pull Request 2452`_
+* Add the trackhub composite datatype
+  (thanks to `@remimarenco <https://github.com/remimarenco>`__.)
+  `Pull Request 2348`_, `Pull Request 2646`_, `Pull Request 2701`_
+* Add the Mothur datatypes
+  (thanks to `@shiltemann <https://github.com/shiltemann>`__.)
+  `Pull Request 2038`_, `Pull Request 2429`_
+* Add the dynamic tool destination tool to Galaxy for defining finer grained
```

Comment:
I moved these 3 to the end of the highlighted section since they are release highlights already